jamon-parent/jamon/src/test/java/org/wicketstuff/jamon/component/JamonAdminPageTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wicketstuff.jamon.component;
+
+import org.apache.wicket.util.tester.WicketTester;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.wicketstuff.jamon.component.JamonAdminPage;
+
+import com.jamonapi.MonitorFactory;
+
+
+public class JamonAdminPageTest
+{
+
+	private WicketTester wicketTester;
+
+	@Before
+	public void beforeEachTest()
+	{
+		wicketTester = new WicketTester(JamonAdminPage.class);
+	}
+
+	@After
+	public void after()
+	{
+		MonitorFactory.getFactory().reset();
+	}
+
+	@Test
+	public void shouldRenderStatisticsPageWithOneMonitor()
+	{
+		JamonTestUtil.startThisManyMonitors(1);
+		wicketTester.startPage(new JamonAdminPage());
+		wicketTester.assertRenderedPage(JamonAdminPage.class);
+	}
+
+	@Test
+	public void shouldRenderStatisticsPageWithTwoMonitors()
+	{
+		JamonTestUtil.startThisManyMonitors(2);
+		wicketTester.startPage(new JamonAdminPage());
+		wicketTester.assertRenderedPage(JamonAdminPage.class);
```

Comment:
Since there is no assertion of the number of the monitors I think there is no need of two test methods.